doc/changes/changes-0.1.0.md

```diff
@@ -0,0 +1,4 @@
+# Exasol Pre-commit Hooks 0.1.0, released 13.08.2020
+
+## Summary 
+initial release
```

Comment:
You should create a ticket "Add initial implementation" and add it here. For any pull request we create a ticket. 

Also, add a link to this file into changelog.md

Reply:
```suggestion
#3 Added initial implementation
```